### PR TITLE
ci: upload .deb files to gemfury

### PR DESCRIPTION
**Pull request by chombourger**
Signed-off-by: Cedric Hombourger <cedric.hombourger@siemens.com>

### CODE REVIEW

**Comment by nejch** [comment]
@bufferoverflow what's your opinion on this? We also have https://github.com/siemens/libuta, https://github.com/siemens/libsecutils and probably many others that could be deployed as .debs but if there is no central solution people will find their own.

https://gitlab.com/groups/gitlab-org/-/epics/6057 (thx @sathieu!) is very close so it may be an option.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -29,3 +29,10 @@ jobs:
           path: |
             build/tmp/deploy/images/nanopi-neo/isar-image-base-debian-bullseye-nanopi-neo.wic.img
             build/tmp/deploy/images/nanopi-neo/isar-image-base-debian-bullseye-nanopi-neo.wic.img.bmap
+      - name: Publish .deb to Gemfury
+        env:
+          GEMFURY_PUSH_TOKEN: ${{ secrets.GEMFURY_PUSH_TOKEN }}
+        run: |
+          find build/tmp/deploy/isar-apt -name '*.deb' -print | while read p; do
```
Comment (vj-kumar) [inline .github/workflows/main.yml:36]: What is the purpose of uploading all debs to gemfury? Sorry, but just new to this.  I am not sure if the intent is to upload only the mtda debs or all the debs as in code. I have no workspace right now, so just not sure what else is there in the isar-apt.

Comment (chombourger) [reply .github/workflows/main.yml:36]: our project was previously using bintray (a light version of artifactory) but that service was sunset. The purpose is to ease installation / update of the mtda packages via `apt-get`. We are uploading all the packages we are building. With this PR, only `all` and `armhf` but `amd64` will be added (to make it really easy to install mtda on your Debian/Ubuntu host)

**Comment by chombourger** [comment]
> @bufferoverflow what's your opinion on this? We also have https://github.com/siemens/libuta, https://github.com/siemens/libsecutils and probably many others that could be deployed as .debs but if there is no central solution people will find their own.
> 
> https://gitlab.com/groups/gitlab-org/-/epics/6057 (thx @sathieu!) is very close so it may be an option.

Our project was using bintray before being moved to the siemens org and before the service was sunset. I agree that it would be good to have a common platform where .deb packages may be uploaded

**Comment by vj-kumar** [comment]
> > @bufferoverflow what's your opinion on this? We also have https://github.com/siemens/libuta, https://github.com/siemens/libsecutils and probably many others that could be deployed as .debs but if there is no central solution people will find their own.
> > https://gitlab.com/groups/gitlab-org/-/epics/6057 (thx @sathieu!) is very close so it may be an option.
> 
> Our project was using bintray before being moved to the siemens org and before the service was sunset. I agree that it would be good to have a common platform where .deb packages may be uploaded

Thanks for the clarification @chombourger

**Review by vj-kumar — APPROVED** [review]

**Review by bikrambhola — APPROVED** [review]

**Pull request merged**